pipelineController.yaml

```
@@ -5,7 +5,11 @@ repositoryTasks:
     phrase:
       - phraseRegex: '.*publish to pypi.*'
         script:
-          - taskName: BuildAndPublish
+          - taskName: BuildAndPublishPyPi
+            definition: jenkins/build_and_publish_pypi.groovy
```

```suggestion
            definition: jenkins/build_and_publish_testpypi.groovy
```

What I would like to see here is having it be impossible to publish a package to the official PyPi via a PR.